connectors/es/sink.py

```diff
@@ -352,6 +381,14 @@ async def run(self, generator, job_type):
         except asyncio.CancelledError:
             self._logger.info("Task is canceled, stop Extractor...")
             raise
+        except ForceCanceledError:
+            self._logger.error(
+                "Extractor cannot be stopped after certain period of time, force cancel the extractor."
+            )
+        except Exception as e:
+            self._logger.critical("Document fetcher failed", exc_info=True)
```

Comment:
```suggestion
            self._logger.critical("Document extractor failed", exc_info=True)
```
Not related, but we can fix this while we're here.